proxy/InkAPI.cc

```diff
@@ -9586,3 +9587,43 @@ TSRegisterProtocolTag(const char *tag)
 {
   return nullptr;
 }
+
+namespace
+{
+// Function that contains the common logic for TSRemapFrom/ToUrlGet().
+//
+tsapi TSReturnCode
```

Comment:
Nitpicky, but not sure this should be tsapi btw. That's for the public interfaces, no?